CONTRIBUTORS-libmv.md

```diff
@@ -1,28 +1,8 @@
----------------
-openMVG authors
----------------
-Pierre Moulon <pmoulon@gmail.com>
-Pascal Monasse <monasse@imagine.enpc.fr>
-Renaud Marlet <renaud.marlet@enpc.fr>
+libmv Contributors
+==================
 
-anhtuann
-Bruno Duisit
-Fabien Castan
-Iivari Äikäs
-luxigo
-Michael Holroyd
-Romain Janvier
-Rory McCann
-Romuald Perrot
-rhiestan
-sergi pujades-rocamora
-sflotron
-vincentweb
-Yohann Salaun
+A subpart of libmv has been included in the "multiview" module and continue to evolve.
```

Comment:
When you say that you have no way to track which function or which file.

Reply:
What do you mean?

Reply:
Since you brutally removed the libmv license in each file we do not have any way to track which file comes from libmv and which one comes from OpenMVG.
Here an example, this file is in multiview https://github.com/alicevision/AliceVision/blob/develop/src/openMVG/multiview/rotation_averaging_l2.hpp#L2 is from OpenMVG but not from libmv.